src/main/scala/com/yotpo/metorikku/metric/MetricReporting.scala

```diff
@@ -18,10 +18,11 @@ class MetricReporting {
         try {
           reportLagTimeColumnUnits match {
             case Some(units) => TimeUnit.valueOf(units) match {
-              case TimeUnit.SECONDS => TimeUnit.SECONDS.toMillis(dataFrame.agg({timeColumn.toString -> "max"}).collect()(0).getLong(0))
-              case TimeUnit.MILLISECONDS => TimeUnit.MILLISECONDS.toMillis(dataFrame.agg({timeColumn.toString -> "max"}).collect()(0).getLong(0))
+              case TimeUnit.MILLISECONDS => TimeUnit.MILLISECONDS.toMillis(dataFrame.agg({timeColumn -> "max"}).collect()(0).getLong(0))
+              case TimeUnit.SECONDS => TimeUnit.SECONDS.toMillis(dataFrame.agg({timeColumn -> "max"}).collect()(0).getLong(0))
+              case _ => dataFrame.agg({timeColumn -> "max"}).collect()(0).getTimestamp(0).getTime()
```

Comment:
```reportLagTimeColumnUnits``` should be specified only if the column is of a LONG type,
so the default case ```.getTimestamp(0)``` will get an exception here.

Reply:
Any suggestions then? I hate that we have these warnings...

Reply:
Throw an exception by default

Reply:
done